auth/controllers.py

```diff
@@ -149,6 +150,8 @@ def _get_token_from_profile(provider, profile, private_key):
     email = norm_profile['email']
     avatar_url = norm_profile['avatar_url']
     user = create_or_get_user(userid, name, username, email, avatar_url)
+    if user.get('new'):
+        action_user('datahub-emails', user)
```

Comment:
the whole point was to avoid having datahub-specific code in the core code-base.

Reply:
But we should call that new method form `auth/extensions.py` somewhere, no? How it's gonna work without that?

Reply:
Ok think got it - the problem is in `datahub-emails` not in the method being called, right?